app/helpers/application_helper/button/server_level_options.rb

```diff
@@ -0,0 +1,10 @@
+class ApplicationHelper::Button::ServerLevelOptions < ApplicationHelper::Button::Basic
+  needs :@record, :@sb
+
+  def visible?
+    x_active_tree == :diagnostics_tree &&
```

Comment:
The point of using the wrapper is that the implementation can be changed and once in a better future we might get rid of `@sb`